sdk/storage/storage-blob/src/utils/utils.common.ts

```diff
@@ -96,34 +96,43 @@ function getProxyUriFromDevConnString(connectionString: string): string {
   return proxyUri;
 }
 
+function getValueInConnString(
+  connectionString: string,
+  argument:
+    | "BlobEndpoint"
+    | "AccountName"
+    | "AccountKey"
+    | "DefaultEndpointsProtocol"
+    | "EndpointSuffix"
+    | "SharedAccessSignature"
+) {
+  const matchCredentials = connectionString.split(";");
```

Comment:
nit: `matchCredentials` doesn't seem to fit here. `components` or `elements`?

Reply:
picked `elements`.